### PR TITLE
Expose saving of YAML templates and hide the legacy UI

### PR DESCRIPTION
**Pull request by kalikiana**
- Expose Save button in the UI
- Hide legacy UI when YAML is/was stored
- Handle conflicts via the `reference` feature (no diff here, just a simple error)

Screenshot from 2019-06-18 11-21-54

Prerequisite: #2103 

Fixes: [poo#54146](https://progress.opensuse.org/issues/54146)

### CODE REVIEW

File: templates/admin/job_template/index.html.ep [hunk 1]
```diff
@@ -15,12 +15,20 @@
         false
     % }
     ;
+    % if ($group->template) {
+    toggleTemplateEditor();
```
Comment (kraih) [inline templates/admin/job_template/index.html.ep]: Indentation looks wrong.

Comment (kalikiana) [reply templates/admin/job_template/index.html.ep]: My editor seems to get confused with the mix... should be corrected now, also another instance at the bottom with HTML

**Comment by okurz** [comment]
Could you deploy that to our staging instances?

**Comment by Martchus** [comment]
Before we allow users to save YAML (and add parameters) dump_templates and load_templates should support them (or at least just ignore them).

**Pull request merged**